web/src/components/core/InstallationFinished.jsx

```diff
@@ -26,60 +26,92 @@ import {
   EmptyStateBody,
   EmptyStateHeader,
   EmptyStateIcon,
+  ExpandableSection,
+  Hint,
+  HintBody,
 } from "@patternfly/react-core";
 
-import { Page } from "~/components/core";
-import { Center, Icon } from "~/components/layout";
+import { If, Page } from "~/components/core";
+import { Icon } from "~/components/layout";
 import { useInstallerClient } from "~/context/installer";
+import { EncryptionMethods } from "~/client/storage";
 import { _ } from "~/i18n";
 
+const TpmHint = () => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Hint className="tpm-hint">
+      <HintBody>
+        <ExpandableSection
+          isExpanded={isExpanded}
+          onToggle={() => setIsExpanded(!isExpanded)}
+          toggleText={
+            <strong>
+              {_("Make sure to boot directly to the new system to finish the configuration of TPM-based decryption.")}
+            </strong>
+          }
+        >
+          <Text>
+            {_("The final step to configure the TPM to automatically open devices will take place during the first boot of the new system. For that to work, the machine needs to boot directly to the new boot loader. If a local media was used to run this installer, make sure is removed before next boot.")}
```

Comment:
NP: I am not sure if the tip is clear enough. I mean "Make sure to boot directly to the new system ...". Me as user can ask "why am I not going to boot directly? I only have to reboot".

I would try to directly indicate an action, for example: "If an installation media was used for running this installer , then remove it before rebooting. Otherwise the TPM-based decryption will not be correctly configured".

And the details would explain that TPM requires to directly boot the new system to finish the configuration. And maybe it could indicate what happens if that is not done.